docs/reference/mapping/removal_of_types.asciidoc

```diff
@@ -41,7 +41,7 @@ field, so documents of different types with the same `_id` could exist in a
 single index.
 
 Mapping types were also used to establish a
-/guide/en/elasticsearch/reference/5.4/mapping-parent-field.html[parent-child relationship]
+https://www.elastic.co/guide/en/elasticsearch/reference/5.4/mapping-parent-field.html[parent-child relationship]
```

Comment:
I think it whould be better to use the asciidoc syntax for internal cross-references (<<url,link text>>) so we keep linking the current version etc... The page that the hard link refers to has the section title `mapping-parent-field`, so the link should be <<mapping-parent-field,parent-child relationship>>